src/main/java/net/daboross/bukkitdev/skywars/config/TranslationsConfiguration.java

```diff
@@ -39,6 +39,29 @@
 
 public class TranslationsConfiguration implements SkyTranslations {
 
+    private static final String MESSAGES_FILE_HEADER = "### messages.yml ###\n"
```

Comment:
Sorry I didn't look over the other pull request! This seems to be an unrelated change, and does obscure the code.